app/store/migrations.js

```diff
@@ -300,6 +300,32 @@ export const migrations = {
     };
     return state;
   },
+  12: (state) => {
+    const {
+      allCollectibles,
+      allCollectibleContracts,
+      ignoredCollectibles,
+      ...unexpectedCollectiblesControllerState
+    } = state.engine.backgroundState.CollectiblesController;
+    state.engine.backgroundState.NftController = {
+      ...unexpectedCollectiblesControllerState,
+      allNfts: allCollectibles,
+      allNftContracts: allCollectibleContracts,
+      ignoredNfts: ignoredCollectibles,
+    };
+    delete state.engine.backgroundState.CollectiblesController;
+
+    state.engine.backgroundState.NftDetectionController =
+      state.engine.backgroundState.CollectibleDetectionController;
+    delete state.engine.backgroundState.CollectibleDetectionController;
+
+    state.engine.backgroundState.PreferencesController.useNftDetection =
+      state.engine.backgroundState.PreferencesController.useCollectibleDetection;
+    delete state.engine.backgroundState.PreferencesController
+      .useCollectibleDetection;
+
+    return state;
+  },
 };
 
 export const version = 11;
```

Comment:
Oops, I think I missed this!
```suggestion
export const version = 12;
```

Reply:
Oops, escaped me as well! Thanks for backing here!